<launch>
  <node pkg="ultraspeech" name="subjectUI" type="subjectUI.py" args="/home/robotlab/ros/ua-ros-pkg/ua_experimental/ultraspeech/teststims.txt 5"/>

  <node pkg="ultraspeech" name="ros_dvgrab" type="ros_dvgrab"/>

  <machine name="shapur" address="shapur" ros-root="/Users/robotlab/ros/ros" ros-package-path="/Users/robotlab/ros">
  </machine>
  <node machine="shapur" pkg="ua_audio_capture" name="audio_capture" type="audio_capture" args="4 96000"/>


</launch>
